src/configuration/ConfigurationManager.ts

```diff
@@ -86,36 +90,13 @@ export class ConfigurationManager {
 }
 
 const getComissionRates = async (): Promise<KeyCommissionRate> => {
-    const path = `${__dirname}/../data/.negotiatedFees`;
-
-    if (fs.existsSync(path)) {
-        const commissionRates: KeyCommissionRate = {};
-
-        console.log('Found .negotiatedFees file. Using Payor Specific Commission Rates.');
-
-        const raw = fs.readFileSync(path, 'utf-8');
-
-        const rows = raw.split(/\r?\n/).filter(row => row);
-
-        rows.forEach((x, index) => {
-            const [key, rate] = x.split('|');
-
-            const takeRate = Number.parseFloat(rate);
 
-            const result = validateCommission(key, takeRate, index);
+    const commissionRates: KeyCommissionRate = {};
 
-            if (!result.isValid) {
-                console.log(result.error);
-                throw new Error(result.error);
-            }
-
-            commissionRates[key] = { commissionRate: takeRate };
-        });
-
-        return commissionRates;
-    }
-
-    return {};
+    getComissionRatesFromFile('knownNegotiatedFees', commissionRates);
```

Comment:
I'm still unclear as to why this file is necessary / useful.

Reply:
And would rather see it named mfNegotiatedFees if that is what it is ... but I think it should be removed unless there is a use case that hasn't been discussed.

Reply:
This has been removed now